mackup/applications/aerospace.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = aerospace
+
+[configuration_files]
+.aerospa ce.toml
```

Comment:
Was this tested? There seems to be a space in the file name.

Reply:
Interestingly, it was not the normal space character but instead unicode U+00a0. 🤷🏼‍♂️